recipe/meta.yaml

```diff
@@ -18,6 +18,7 @@ requirements:
     - __unix
     - pip
     - python >=3.7
+    - setuptools
```

Comment:
The linter asked us to add this (previously unstated) build dependency.

I don't think setuptools is a dependency of pip any more (or won't be in the near future) so we need to explicitly specify this dependency (or get on with PEP517).